### PR TITLE
Feature/offset

### PR DESCRIPTION
**Pull request by sdahlbac**
Use real OFFSET instead of ROW_NUMBER() emulation

### CODE REVIEW

File: lib/sequel/jdbc/as400.rb [hunk 1]
```diff
@@ -59,7 +59,6 @@ def begin_transaction(conn, opts = OPTS)
 
       # Dataset class for AS400 datasets accessed via JDBC.
       class Dataset < JDBC::Dataset
-        include EmulateOffsetWithRowNumber
```
Comment (olleolleolle) [inline lib/sequel/jdbc/as400.rb]: is `EmulateOffsetWithRowNumber` used after this?

Comment (sdahlbac) [reply lib/sequel/jdbc/as400.rb]: Not as far as I can see, (and it shouldn't since we don't want that behavior)

Comment (sdahlbac) [reply lib/sequel/jdbc/as400.rb]: ah, you are asking if the require is still needed

**Comment by perlun** [comment]
**NB**: There _might_ be DB2/iSeries versions where this does not work. If/when we discover such environments, we or other users of this gem will have to fix this to support them.

**Pull request merged**